### ISSUE TITLE
Stuck loading into singleplayer

### ISSUE THREAD

**Issue by NillerMedDild** [TerraForged/tracker#142]
Attempting to load into a singleplayer world with TerraForged generation frequently gets stuck on loading terrain some %
It has happened 4 out of 5 tries today. This has been an issue for a long time, but it has never been this frequent.
It seems to be happen more frequently if the window is not in focus, that might be a coincidence though.

I had help analyzing the VisualVM snapshot and was told `Terraforged is waiting on a chunk... no threads left that are generating chunks though`

## Logs
VisualVM snapshot
[stuck-loading-singleplayer.zip](https://github.com/TerraForged/tracker/files/6931254/stuck-loading-singleplayer.zip)

Latestlog
[latest.log](https://github.com/TerraForged/tracker/files/6931251/latest.log)

### Versions
- TerraForged-1.16.5-0.2.13
- Forge-1.16.5-36.2.1
- Enigmatica 6 Modpack 0.5.8-in_dev

**Comment by Won-Ton** [comment]
Could you provide PC specs + perf config please. Is TF's watchdog enabled also?

**Comment by NillerMedDild** [comment]
PC specs:
```
GPU          NVIDIA GeForce GTX 1060
CPU          Intel(R) Core(TM) i5-7300HQ CPU @ 2.50GHz
RAM	16,0 GB
```

I don't know whether the TF watchdog is enabled, how do I check?

**Comment by Won-Ton** [comment]
Thanks. It's in general.conf. The deadlock timeout should be greater than zero (60000 (1 minute) is the default). Suspect it may not detect anything though.

**Comment by NillerMedDild** [comment]
Ah yes, that is enabled. Doesn't seem like it detects anything to be honest.

```toml
# The version of this config. Do not edit (it'll wipe your settings!)
version = "1.1"
# Set the preset to use when creating a new world.
default_preset = "Enigmatica"
# Set whether tooltips should be displayed by default in the config gui.
tooltips = true
# Set whether coordinates should be displayed by default in the config gui.
coords = true
# Set whether multiple threads should be used to perform structure searches.
# This can greatly increase the speed of finding structures via the /locate command.
async_structure_search = true
# Set the number of milliseconds that a structure search can run for before it is aborted.
structure_search_timeout = 5000
# The number of milliseconds a single feature/structure can generate for before a warning
# is printed to the logs. This may help track down mods that are causing world-gen to run slow.
# Set to -1 to disable.
feature_warn_time = 100
# The number of milliseconds after which the server will be considered 'deadlocked' (when it
# gets stuck trying to generate a feature/structure). This is usually caused by third-party mods.
# Set to -1 to disable deadlock detection & reporting (the game may freeze indefinitely without it).
chunkgen_timeout = 60000
# Fixes a vanilla serialization bug for block tags which TerraForged relies on
# to slightly increase the vanilla ore distribution to suit the taller terrain.
# Disable if this causes issues with other mods.
fixBlockTags = true
```

**Comment by NillerMedDild** [comment]
Fixed as a part of #141 

9 / 10 new TF worlds started successfully with the patched version. Thanks for the fix!
In the 1 that did not start, the cause was not TerraForged.
